bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -115,14 +109,20 @@ spec:
           name: host-cni-netd
         - mountPath: /var/lib/cni/networks/openshift-sdn
           name: host-var-lib-cni-networks-openshift-sdn
+        - mountPath: /lib/modules
+          name: host-modules
+          readOnly: true
+        - mountPath: /etc/sysconfig/openshift-sdn
+          name: etc-sysconfig-openshift-sdn
+          readOnly: true
         resources:
           requests:
             cpu: 100m
             memory: 200Mi
         env:
         - name: OPENSHIFT_DNS_DOMAIN
           value: cluster.local
-        - name: NODENAME
+        - name: K8S_NODE_NAME
```

Comment:
how does this work exactly? I don't see anything in origin, kubernetes, or cluster-network-operator that looks at `K8S_NODE_NAME`

Reply:
It only merged a few days ago ;-). https://github.com/openshift/origin/blob/master/pkg/cmd/openshift-sdn/cmd.go#L145